src/Common/tests/System/PlatformDetection.cs

```diff
@@ -158,9 +158,13 @@ public static bool IsWinRT
         public static bool IsWindowsSubsystemForLinux => m_isWindowsSubsystemForLinux.Value;
         public static bool IsNotWindowsSubsystemForLinux => !IsWindowsSubsystemForLinux;
 
-        public static bool IsNotFedoraOrRedHatOrCentos => !IsDistroAndVersion("fedora") && !IsDistroAndVersion("rhel") && !IsDistroAndVersion("centos");
+        public static bool IsNotFedoraOrRedHatFamily => !IsDistroAndVersion("fedora") && !IsRedHatFamily;
 
         public static bool IsFedora => IsDistroAndVersion("fedora");
+        public static bool IsRedHatFamily => IsRedHatFamilyAndVersion();
```

Comment:
please add comment that this is cover RedHat and Centos. Just in case if anyone want to check Centos and don't know it is covered by this property.

Reply:
pending